dsnp-wallet/dsnp-wallet/Source/Modules/Onboarding/SeedPhrase/SeedPhraseViewController.swift

```diff
@@ -104,11 +105,18 @@ class SeedPhraseViewController: UIViewController {
     
     private func setLabels() {
         titleLabel.text = state == .viewSeedPhrase ? "There will be a test" : "This is the test"
-        
         descriptionLabel.numberOfLines = 0
         descriptionLabel.text = state == .viewSeedPhrase ? "Please carefully write down these 12 words below in numerical order (1-12) and store your secret phrase securely. You will confirm them in order on the next screen." : "Tap the words in numerical order (1-12) to verify your recovery phrase"
     }
     
+    private func setAccessibilityIds() {
+        titleLabel.accessibilityIdentifier = "pageTitle"
+        descriptionLabel.accessibilityIdentifier = "pageDescriptionLabel"
+        topCollectionView?.accessibilityIdentifier = "selectionGridResult"
+        bottomCollectionView?.accessibilityIdentifier = "selectionGridInput"
+    }
+    
+    
```

Comment:
Very minor thing, but could you check to see if this is double spaced or single? Just want to try to keep everything format uniform.

Reply:
Removed 1 space